### PR TITLE
Dollar PnL was NaN for user on positions page and positions card

### PR DESCRIPTION
**Pull request by EstherSeyi**
# Task: Show the user's dollar PnL

## Description

This PR fixes the issue of dollar pnl showing nan for a user on position page and card

Fixes # (issue)
#95 
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

You can test this PR by ensuring unrealized pnl on position card and position page is not nan

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Review by alexisgauba — APPROVED** [review]

File: packages/frontend/src/lib/pnl.ts [hunk 1]
```diff
@@ -7,7 +7,14 @@ type ShortPnLParams = {
...
 
 export function calcUnrealizedPnl({ wethAmount, buyQuote, ethPrice }: ShortPnLParams) {
-  if (wethAmount.isEqualTo(0) || buyQuote.isEqualTo(0) || ethPrice.isEqualTo(0)) {
+  if (
+    wethAmount.isEqualTo(0) ||
+    !wethAmount.isFinite() ||
+    buyQuote.isEqualTo(0) ||
+    !buyQuote.isFinite() ||
+    ethPrice.isEqualTo(0) ||
+    !ethPrice.isFinite()
```
Comment (yichiehliu) [inline packages/frontend/src/lib/pnl.ts:16]: Just wondering would the ethprice be finite?

Comment (EstherSeyi) [reply packages/frontend/src/lib/pnl.ts:16]: it may never be but .isFinite covers for the number not being NaN too

Comment (yichiehliu) [reply packages/frontend/src/lib/pnl.ts:16]: Maybe using isNaN would be more accurate?

Comment (EstherSeyi) [reply packages/frontend/src/lib/pnl.ts:16]: I think .isFinite covers more cases

Comment (yichiehliu) [reply packages/frontend/src/lib/pnl.ts:16]: kk gotcha

**Review by yichiehliu — APPROVED** [review]

**Pull request merged**